core/src/mindustry/ui/dialogs/ResearchDialog.java

```diff
@@ -485,10 +527,10 @@ protected void updateRelative(float percentDelta){
                 button.userObject = node.node;
                 button.setSize(nodeSize);
                 button.update(() -> {
-                    button.setDisabled(net.client() && !mobile);
+                    button.setDisabled(false && !mobile);
```

Comment:
```suggestion
button.setDisabled(!mobile)
```
I dont understand the reasoning for false here...

If you never want the button to be disabled then
```suggestion
```